tests/consensus_tests/test_resharding_abort_live_lock.py

```
@@ -0,0 +1,34 @@
+
+        migrate_points(
+            peer_uris[0],
+            info["peer_id"],
+            local_shard["shard_id"],
+            target_shard["peer_id"],
+            target_shard["shard_id"],
+            "down",
+        )
+
+    # Kill last peer
+    processes.pop().kill()
```

Ruff (0.14.10)

`processes` may be undefined, or defined from star imports

(F405)